.env.example

```diff
@@ -1,5 +1,15 @@
+ENVIRONMENT=local
 API_CLIENT_ID=approved-premises-ui
 API_CLIENT_SECRET=clientsecret
-COMMUNITY_ACCOMMODATION_API_SERVICE_NAME=approved-premises
+SERVICE_SIGNIFIER=path
+APPROVED_PREMISES_SUBDOMAIN=approved-premises
+TEMPORARY_ACCOMMODATION_SUBDOMAIN=temporary-accommodation
 APPROVED_PREMISES_ROOT_PATH=approved-premises
 TEMPORARY_ACCOMMODATION_ROOT_PATH=temporary-accommodation
+COMMUNITY_ACCOMMODATION_API_SERVICE_NAME=temporary-accommodation
```

Comment:
What would you think of getting rid of this from .env.example, and updating the default in config, which is `approved-premises`?